index.rst

```diff
@@ -109,66 +137,54 @@ syntactically equivalent to a semantic version
-semver-compatible, and ``rest`` is transformed as follows: underscores
-become dots, and any nonalphanumeric characters remaining are simply
-dropped.
+semantic version.
+The information in ``cycle`` is already semver-compatible, and ``rest`` is transformed as follows: underscores become dots, and any nonalphanumeric characters remaining are simply dropped.
+
+Sort priority as implemented by the Nublado controller therefore does **not** follow semver, as it does take the build field into account when sorting, which is not permitted under semver.
  
 Alias Tags
 ----------
 
-Alias tags are exactly those tags we do not expect to be stable.  There
-are currently two items in this category: the recommended tag, and
+Alias tags are exactly those tags we do not expect to be stable.
+There are currently two items in this category: the recommended tag, and
 ``latest``, which is defined by Docker as special and is the tag-of-last
 resort: if a docker image is specified without a tag, its tag is
-implicitly ``latest``.  Our builds include ``latest_weekly``,
-``latest_daily``, and ``latest_release`` tags, but Data Management does
-not make use of them in the spawner options form.
+implicitly ``latest``.
+Our builds include ``latest_weekly``, ``latest_daily``, and ``latest_release`` tags, but Data Management does not make use of them in the spawner options form.
 
 The "recommended" tag is a singleton: it is always an alias pointing
 to the image that, at any moment in time, we believe most users should
-be using.  Conventionally, this will be a recent weekly build, and
-conventionally it has been ``recommended``.  It can be any arbitrary
-string, and this will be useful, for example, to allow different
-recommended images in different environments (e.g. ``recommended_idf``
-versus ``recommended_summit``).
-
-Alias tags are passed in to the tag parsing machinery from the outside
-as a list of strings.  Only a tag that is an exact string match to one
-of those strings will be categorized as an alias tag.  If it is set, the
-"recommended" tag will always be prepended to the alias tag list.  If
-the alias tag list does not include ``latest``, even though that is by
-its nature an alias tag, the ``latest`` image will not be prepulled.
+be using at a given RSP instance (it may, and does, differ between instances).
+Conventionally, this has historically been a recent weekly build, but in operations will generally be a recent release build.
+Conventionally this tag has been the literal string ``recommended``.
+It can be any arbitrary string.
+
+Alias tags are passed in to the tag parsing machinery from the outside as a list of strings.
+Only a tag that is an exact string match to one of those strings will be categorized as an alias tag.
+If it is set, the "recommended" tag will always be prepended to the alias tag list.
+If the alias tag list does not include ``latest``, even though that is by its nature an alias tag, the ``latest`` image will not be prepulled.
```

Comment:
This isn't true, though, right? If the `latest` tag points to an image that's prepulled, it will be prepulled even if it's not listed in the `aliasTags` list.